week2/todoList/todo.js

```diff
@@ -0,0 +1,86 @@
+function useTodoList({ container, input, submit, deleteBtn }) {
+  const onAdd = (event) => {
+    // event.target => span 아이콘
+    let parentContainer =
+      event.target.parentElement.parentElement.parentElement;
```

Comment:
parentElement를 여러개 써도 되지만, Closest 메서드를 이용해서 간결하게 적어도 되지 않을까요!!

해당 메서드는 가장가까운 조상을 반환하는  메서드입니다!! 

```suggestion
      let AddBtn = event.target
      let parentContainer = addBtn.closest('.main-section'); // 누른 요소와 가장 가까운 부모요소를 반환해줘요!! 
```